### ISSUE TITLE
关于运行时jar文件

### ISSUE THREAD

**Issue by binmu123** [Tencent/angel#111]
你好，
angel提供两种方式运行本地和yarn模式，我查看了bin下的angel-submit和angel-example两个启动脚本，实际上这里就是环境变量的设置添加ANGEL_HOME和classpath，但是我运行不起来，因为jar包没有加入CLASSPATH路径，看脚本jar存在的路径为"$bin"/../lib   但是我没有找到lib文件夹，所以无法将jar配置在CLASSPTH路径中。我查看项目也没有找到相应的jar文件。
请问下jar在哪个位置？需要我自己生成jar文件，然后运行吗？

**Comment by paynie** [comment]
Thanks for reporting this issue.  Do you mean that the `lib` directory is  not included in `angel-1.0.0-bin.zip`?

**Comment by binmu123** [comment]
Yes,exactly.
I find release,there is only Source code, and no angel-1.0.0-bin.zip, still does not exist lib.
Maybe it needs to compile by yourself. so I just follow the Angel compiler guide to compile.
During Angel recompilation, in the translation of the time there is a problem, the document said the need for Protobuf> = 2.5.0, but I just started using 2.5.0 when the error, Reported the need for Protobuf need 2.6.1, I do not know the problem is maven? I continued to compile after 2.6.1 and then succeeded.I do not know if the document is wrong. End,the problem is solved.
thanks for your answer.
